static/app/views/monitors/components/overviewTimeline/gridLines.tsx

```diff
@@ -33,50 +34,67 @@ interface Props {
   stickyCursor?: boolean;
 }
 
-/**
- * Aligns a date to a clean offset such as start of minute, hour, day
- * based on the interval of how far each time label is placed.
- */
-function alignTimeMarkersToStartOf(date: moment.Moment, timeMarkerInterval: number) {
```

Comment:
Moved to utils to colocate with the timeWindowConfig computation function that also has some clamping logic.